README.md

```diff
@@ -1,12 +1,6 @@
-# React + Vite
+App Name: To Do List
+Description: This application will allow a user to add, modify, and delete tasks to be completed.
 
-This template provides a minimal setup to get React working in Vite with HMR and some ESLint rules.
+Installation instructions:
```

Comment:
Not an issue for right now, but for whichever project you ever make public, I would recommend putting the steps here so users can look at what you've been working on :)